sessions/real-world-forecasts.qmd

```diff
@@ -0,0 +1,796 @@
+The short-term forecasts were published on the US CDC website as the official forecasts of record from the federal government.
+
+During this time, the project collected over 9,000 forecast submissions from 130 models. 
+The project collected nearly 1 billion rows of model output data, and has led to multiple research papers on epidemiological forecasting.[@ray_comparing_2023;@cramer_evaluation_2022;@fox_optimizing_2024]
+
+The project paused forecasting during the 2024 summer, and restarted in late fall 2024 under the purview of the US CDC Centers for Forecasting and Outbreak Analytics (CFA). 
+When it re-started, the hub was launched using hubverse-style forecast guidelines.
+[The public dashboard for the COVID-19 Forecast Hub](https://reichlab.io/covidhub-dashboard/) has an up-to-date record of the forecasts submitted.
+
+In this session, we will access and analyse forecasts from the CFA-run COVID-19 Forecast Hub.
+
+
+## Forecast dimensions
+
+Here are some details about the structure of the COVID-19 Forecast Hub project. 
+Forecasts are collected every week, with submissions due on Wednesdays.
+Teams make predictions of -1 through 3 week ahead counts of new hospital admissions due to COVID-19. 
+The ground truth data ("target data" in hubverse lingo) is taken from [the Weekly Hospital Respiratory Data (HRD) Metrics by Jurisdiction, National Healthcare Safety Network (NHSN) dataset](https://data.cdc.gov/Public-Health-Surveillance/Weekly-Hospital-Respiratory-Data-HRD-Metrics-by-Ju/mpgq-jmmr/about_data).
+Teams may forecast for up to 53 locations, including the US national level, all 50 states, plus Puerto Rico and Washington DC.
+Forecasts are solicited in a quantile format.
+Further details about the hub are available at its [GitHub README page](https://github.com/CDCgov/covid19-forecast-hub/blob/main/README.md).
+
+# The forecasts
+
+We will start by accessing forecast data from the COVID-19 Forecast Hub. 
+Because the hub takes advantage of hubverse cloud data storage architecture, both the forecast and target data are stored in an S3 bucket that can be accessed using the `hubData` R package.
+
+::: callout-tip
+For a complete list of hubs (many of which have public data available) check out the hubverse [list of hubs](https://hubverse.io/community/hubs.html).
+:::
+
+
+## Accessing forecast data from the cloud
+
+The following code can be used to retrieve covid forecasts from the cloud. 
+However, there are over 2 million rows of data, and it can take some bandwidth to download.
+To make these results more reproducible and to provide a snapshot of the data, we also provide the forecasts as a data object that can be loaded directly from the `nfidd` package.
+
+```{r load-forecasts, eval=FALSE}
+hub_path_cloud <- hubData::s3_bucket("s3://covid19-forecast-hub")
+hub_con_cloud <- hubData::connect_hub(hub_path_cloud, skip_checks = TRUE)
+
+covid_forecasts <- hub_con_cloud |> 
+  dplyr::filter(
```

Comment:
Is there a reason to switch to package::function style partly here and elsewhere?